doc/transform_methods/rbasex-sim.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import division, print_function
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from abel.tools.analytical import SampleImage
+from abel.tools.vmi import Ibeta
+from abel.rbasex import rbasex_transform
+
+rmax = 200
+scale = 10000
+
+vlim = 0.5
+pvlim = 0.5 * np.sqrt(rmax)
+
+def rescaleI(im):
+    return np.sqrt(np.abs(im)) * np.sign(im)
+
+# test distribution
+source = SampleImage(n=2 * rmax - 1).image / scale
+Isrc, _ = Ibeta(source)
+Inorm = (Isrc**2).sum()
+
+# simulated projection fith Poissonian noise
+proj, _ = rbasex_transform(source, direction='forward')
+proj[proj < 0] = 0
+proj = np.random.RandomState(0).poisson(proj)  # (reproducible, see NEP 19)
```

Comment:
NumPy Enhancement Proposal 19 — Random Number Generator Policy:
https://numpy.org/neps/nep-0019-rng-policy.html
This link is probably stable, so I can include it instead of “NEP 19”.

Reply:
Cool!